metrics/src/main/java/ai/vespa/metrics/Unit.java

```diff
@@ -81,13 +81,14 @@ private enum BaseUnit {
         CONNECTION("connection"),
         DOCUMENT("document"),
         DOCUMENTID("documentid"),
+        DOLLAR("dollar"),
```

Comment:
I'm wondering if "dollar" is ambiguous (AUSD, CAD), so I suggest to use 'USD' instead.

Reply:
I think it's ok ...